products/bootstrap.md

```diff
@@ -31,6 +32,7 @@ releases:
     eol: false
     latest: "5.3.3"
     latestReleaseDate: 2024-02-20
+    extendedSupport: false
```

Comment:
For all releases, the `extendedSupport` field must be moved just after the `eol` field (see https://github.com/endoflife-date/endoflife.date/wiki/Style-Guide).

Reply:
Thanks for pointing that out! Done [198ec39](https://github.com/endoflife-date/endoflife.date/pull/4859/commits/198ec39ef5e895a5f3a2298a18294eddcae54747)